src/components/NotificationModal/NotificationModal.js

```diff
@@ -21,9 +28,10 @@ const NotificationModal = (props) => {
       size="small"
       footer={footer}
       dismissible
-      {...props}
```

Comment:
I am not sure that it is correct to delete `{...props}`.
Previously `props` contained `onClose` function which was passed to `Modal` component.
But now `{...rest}` does not contain `onClose`.
I suppose that `onClose` is necessary for `Modal`.

Reply:
@artem-blazhko it is decomposed into the component. So, if the `rest` of the props contains `onClose`, the `Modal` will receive it.

Reply:
@artem-blazhko thank you for noticing, you are correct